sktime/forecasting/model_selection/_split.py

```diff
@@ -792,8 +758,7 @@ def get_cutoffs(self, y: Optional[ACCEPTED_Y_TYPES] = None) -> np.ndarray:
         """
         if array_is_int(self.cutoffs):
             return check_cutoffs(self.cutoffs)
-        else:
-            return np.argwhere(y.index.isin(check_cutoffs(self.cutoffs))).flatten()
+        return np.argwhere(y.index.isin(check_cutoffs(self.cutoffs))).flatten()
```

Comment:
why?

Reply:
same as in the other comment (https://github.com/sktime/sktime/pull/4108#discussion_r1070364781)